recipe/meta.yaml

```diff
@@ -1,38 +1,36 @@
 {% set name = "pyprojroot" %}
-{% set version = "0.2.0" %}
+{% set version = "0.3.0" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
+  name: {{ name|lower }}
+  version: {{ version }}
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
-  sha256: "a79900dc52ee097bfd8d917a3d45e2b98494c47a57ba3a71bf609d7a156732e8"
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 109705bb790968704958efcfc5ccce85d8e3dafa054897cc81371fcbbf56cb10
 
 build:
-  number: 0
-  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 1
```

Comment:
build number should still be 0

Reply:
Thank you! Just merged my commits to one... and here we go.